app/views/settings/_advanced_messenger_settings.html.erb

```diff
@@ -13,4 +13,9 @@
     <%= hidden_field_tag 'settings[disable_fix_for_scroll_to_anchor]', 0, :id => nil %>
     <%= check_box_tag 'settings[disable_fix_for_scroll_to_anchor]', 1, Setting.plugin_redmine_advanced_messenger[:disable_fix_for_scroll_to_anchor] == '1' %>
     <em class="info"><%=raw l(:disable_fix_for_scroll_to_anchor_note) %></em>
+</p>
+<p>
+    <label for="settings_notification_mail_content"><%= l(:notification_mail_content) %></label>
+    <%= select_tag 'settings[notifications_mail_option]', options_for_select([['Teaser', 'teaser'], ['Whole note', 'whole_note']], Setting.plugin_redmine_advanced_messenger[:notifications_mail_option]), include_blank: false %>

```

Comment:
I think `Teaser` and `Whole note` needs internationalization too.

Reply:
Good catch! I added internationalization for "Teaser" and "Whole note" after I saw your comment. Thanks for pointing that out!